src/styles/index.scss

```
@@ -1,3 +1,96 @@
+@import "./variables";
+@import "./stars";
+
 body {
   margin: 0;
+  font-family: $main-font;
+  color: $main-color;
+  font-size: 12px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__photo {
+    margin: 32px 19px 0;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__element {
+    margin: 37px 16px 0;
+  }
+
+  &__title {
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+    margin-top: 0;
+    margin-bottom: 16px;
+  }
+
+  &__rewievs {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__rewievsTitle {
```

```suggestion
  &__rewievs-title {
```
you should follow bem naming conventions